### PR TITLE
Potential readme modifications (documentation-only PR)

### PR DESCRIPTION
**Pull request by JospaUF**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

This pull request is a documentation-only pull request. I only made potential changes to the readme file by adding a Table of Contents, some extra information in certain sections of the readme to make it more clear and easy to understand, and some potential add-ons at the end of the readme file that would need to be filled in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,22 +9,41 @@ Rizin is portable and it can be used to analyze binaries, disassemble code,
...
+for the supported operating system.
 
 # How to build
 
 Use `meson` to compile and install Rizin. Please make sure to get an updated
 `meson` (e.g. get it with `pip install meson` if your system does not provide
 one that is at least version `0.55.0`).
 
-Clone this repository:
+Fork the repository, then clone the repository:
```
Comment (XVilka) [inline README.md:44]: Why fork? Remove this change

File: README.md [hunk 2]
```diff
@@ -125,3 +152,9 @@ We also provide the following partial bridges to other messaging platforms:
 [test/README.md]: https://github.com/rizinorg/rizin/blob/dev/test/README.md
 [BUILDING.md]: https://github.com/rizinorg/rizin/blob/dev/BUILDING.md
 [DEVELOPERS.md]: https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md
+
```
Comment (XVilka) [inline README.md:155]: Unnecessary

File: README.md [hunk 3]
```diff
@@ -102,11 +126,14 @@ useful for shell scripting or as separate standalone tools:
...
 
+To learn more about each tool and their usages, take a look at the 
+[official Rizin book](https://book.rizin.re) and scroll down the table of contents until you find the "Command Line Tools" section.
+
 ## Scripting
 
 We provide a way to interact with Rizin from Python, Haskell, OCaml,
 Ruby, Rust, and Go languages through [rzpipe](https://github.com/rizinorg/rz-pipe).
-Other languages although not currently supported could be easily added.
+Other languages although not currently supported could be easily added. Interact with the community if you would like to see other languages!
```
Comment (XVilka) [inline README.md:136]: Unnecessary

File: README.md [hunk 4]
```diff
@@ -45,18 +64,17 @@ $ rizin
 To uninstall rizin, execute `sudo ninja -C build uninstall`.
 
 
-Please have a look at [BUILDING.md][] for more information about building Rizin.
+Please have a look at the [BUILDING.md][] for more information about building Rizin and for alternative methods.
```
Comment (XVilka) [inline README.md:67]: Remove "alternative methods"

File: README.md [hunk 5]
```diff
@@ -9,22 +9,41 @@ Rizin is portable and it can be used to analyze binaries, disassemble code,
...
 
 Use `meson` to compile and install Rizin. Please make sure to get an updated
 `meson` (e.g. get it with `pip install meson` if your system does not provide
 one that is at least version `0.55.0`).
 
-Clone this repository:
+Fork the repository, then clone the repository:
 ```
-$ git clone https://github.com/rizinorg/rizin
+$ git clone "url of the forked repo"
```
Comment (XVilka) [inline README.md:46]: Remove

File: README.md [hunk 6]
```diff
@@ -9,22 +9,41 @@ Rizin is portable and it can be used to analyze binaries, disassemble code,
 debug programs, as a forensic tool, as a scriptable command-line hexadecimal
 editor able to open disk files, and much more!
 
-To learn more on Rizin you may want to read the
-[official Rizin book](https://book.rizin.re).
+To learn more on Rizin, read the
+[official Rizin book](https://book.rizin.re). 
+
+# Table of Contents
```
Comment (XVilka) [inline README.md:15]: I am not sure if TOC makes sense for README, especially such a small one.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (wargio) [reply README.md:15]: this is easy to break. i'm against this.

File: README.md [hunk 7]
```diff
@@ -125,3 +152,9 @@ We also provide the following partial bridges to other messaging platforms:
 [test/README.md]: https://github.com/rizinorg/rizin/blob/dev/test/README.md
 [BUILDING.md]: https://github.com/rizinorg/rizin/blob/dev/BUILDING.md
 [DEVELOPERS.md]: https://github.com/rizinorg/rizin/blob/dev/DEVELOPERS.md
+
+# Credits
+<!-- add collaborators and team members, linking the github or other types of media -->
+
+# License
+<!-- add a license here if applicable? -->
```
Comment (wargio) [inline README.md:160]: not needed.

**Comment by XVilka** [comment]
I am closing this because no reaction on our feedback. Feel free to open a new PR if/when you will want to do that.

**Pull request closed without merging**